Files/Helpers/ExternalResourcesHelper.cs

```diff
@@ -12,72 +12,72 @@ namespace Files.Helpers
 {
     public class ExternalResourcesHelper
     {
-        public List<AppTheme> Themes = new List<AppTheme>()
+        public List<AppSkin> Skins = new List<AppSkin>()
         {
-            new AppTheme
+            new AppSkin
             {
                 Name = "DefaultSkin".GetLocalized(),
             },
         };
 
-        public StorageFolder ThemeFolder { get; set; }
-        public StorageFolder OptionalPackageThemeFolder { get; set; }
+        public StorageFolder SkinFolder { get; set; }
+        public StorageFolder OptionalPackageSkinFolder { get; set; }
 
-        public string CurrentThemeResources { get; set; }
+        public string CurrentSkinResources { get; set; }
 
-        public async Task LoadSelectedTheme()
+        public async Task LoadSelectedSkin()
         {
-            if (App.OptionalPackageManager.TryGetOptionalPackage(Constants.OptionalPackages.ThemesOptionalPackagesName, out var package))
+            if (App.OptionalPackageManager.TryGetOptionalPackage(Constants.OptionalPackages.SkinsOptionalPackagesName, out var package))
             {
                 Debug.WriteLine(package.InstalledLocation.Path);
-                OptionalPackageThemeFolder = package.InstalledLocation;
+                OptionalPackageSkinFolder = package.InstalledLocation;
             }
 
-            ThemeFolder = await ApplicationData.Current.LocalFolder.CreateFolderAsync("Themes", CreationCollisionOption.OpenIfExists);
+            SkinFolder = await ApplicationData.Current.LocalFolder.CreateFolderAsync("Skins", CreationCollisionOption.OpenIfExists);
```

Comment:
This will mean that any skins the user already has won't be in the new folder. Should we maybe rename the old folder if it exists?

Reply:
I don't believe we shipped support for custom themes yet so I think it's fine to skip renaming the old folder.

Reply:
Support for themes was initially shipped in 1.5. 
![image](https://user-images.githubusercontent.com/59544401/121574706-6685d600-c9db-11eb-8edc-a50f18a150da.png)